fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/mixin/attachment/ServerWorldMixin.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.attachment;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.datafixer.DataFixTypes;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.world.PersistentState;
+
+import net.fabricmc.fabric.impl.attachment.AttachmentPersistentState;
+
+@Mixin(ServerWorld.class)
+abstract class ServerWorldMixin {
+	@Inject(at = @At("TAIL"), method = "<init>")
+	private void createAttachmentsPersistentState(CallbackInfo ci) {
+		// Force persistent state creation
+		ServerWorld world = (ServerWorld) (Object) this;
+		var type = new PersistentState.Type<>(
+				() -> new AttachmentPersistentState(world),
+				nbt -> AttachmentPersistentState.read(world, nbt),
+				DataFixTypes.LEVEL
```

Comment:
This is definitely incorrect. Fortunately, if we declare dependency to Object Builder, its mixin will handle `null` argument.

Reply:
I didn't write this code and frankly I don't know how it works. Why is it broken?